packages/x-date-pickers-pro/src/models/index.ts

```diff
@@ -3,3 +3,5 @@ export * from './fields';
 export * from './range';
 export * from './validation';
 export * from './multiInputRangeFieldClasses';
+
+export type { RangePosition } from '@mui/x-date-pickers/internals';
```

Comment:
We need the type on the community package, but as internal, and we make it public on the pro package